jquery.simulate.js

```diff
@@ -200,8 +200,8 @@ $.extend( $.simulate.prototype, {
 		return event;
 	},
 
-	dispatchEvent: function( elem, type, event ) {
-		if ( elem[ type ] ) {
+	dispatchEvent: function( elem, type, event, options ) {
+		if ( elem[ type ] && !options ) {
```

Comment:
Thanks for the explanation, @dmethvin.

I can't check this in IE8 myself, unfortunately, but the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/dispatchEvent) corroborate that `dispatchEvent` is not compatible versions of IE previous to 9.

Generally though, don't we want to use `dispatchEvent()` by default and use `click()` as the fallback? In other words, should we change the order of those tests?